docs/developers/packages/titus-frontend/README.md

```diff
@@ -85,6 +85,12 @@ To enable it, do the following:
 1. Set `REACT_APP_AUTH_PROVIDER` to `AD` in the **`titus-frontend`** `.env` file.
 3. Change the `render` method in `src/index.js` to the commented out replacement that wraps it in `runWithAdal`.
 
+
+## Authorization example
+_Authentication_ (above) allows your app to verify the identity of your users, while _Authorization_ allows you to manage which resources in your app a user can access.  **`titus-frontend`** provides a an example authz request in `src/components/authz-check` that requests a privileged resource from the backend. 
+
+Refer to **`titus-backend`** [Configure Authorization] where policies specific to your application. 
```

Comment:
did you intend to put a link to the backend config here? not sure if I'm doing anything wrong but the link doesn't seem to work for me.